packages/gatsby-plugin-vanilla-extract/README.md

```diff
@@ -5,7 +5,7 @@ Gatsby plugin which wraps the [vanilla-extract](https://vanilla-extract.style/)
 ## Setup
 
 ```shell
-npm install gatsby-plugin-vanilla-extract @vanilla-extract/babel-plugin @vanilla-extract/css @vanilla-extract/webpack-plugin
+npm install --save-dev gatsby-plugin-vanilla-extract @vanilla-extract/css @vanilla-extract/webpack-plugin
```

Comment:
Just for the sake of consistency:

We tell Gatsby users to install the plugins into dependencies, not devDependencies in all docs. Was it necessary in the context of Gatsby to move the vanilla-extract deps to the devDeps? If not, I'd do it like this:

```suggestion
npm install gatsby-plugin-vanilla-extract @vanilla-extract/css @vanilla-extract/webpack-plugin
```